File: python/Mouse.py
class Mouse:
    def __init__(self, nama="", bentuk_mouse="", DPI="", warna="", jumlah_tombol=0, tipe_koneksi=""):
        self.nama = nama
        self.bentuk_mouse = bentuk_mouse
        self.DPI = DPI
        self.warna = warna
        self.jumlah_tombol = jumlah_tombol
        self.tipe_koneksi = tipe_koneksi
    
    def setNama(self, nama):
        self.nama = nama
    
    def setBentukMouse(self, bentuk_mouse):
        self.bentuk_mouse = bentuk_mouse
    
    def setDPI(self, DPI):
        self.DPI = DPI
    
    def setWarna(self, warna):
        self.warna = warna
    
    def setJumlahTombol(self, jumlah_tombol):
        self.jumlah_tombol = jumlah_tombol
    
    def setTipeKoneksi(self, tipe_koneksi):
        self.tipe_koneksi = tipe_koneksi
    
    def getNama(self):
        return self.nama
    
    def getBentukMouse(self):
        return self.bentuk_mouse
    
    def getDPI(self):
        return self.DPI
    
    def getWarna(self):
        return self.warna
    
    def getJumlahTombol(self):
        return self.jumlah_tombol
    
    def getTipeKoneksi(self):
        return self.tipe_koneksi

File: cpp/Mouse.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class Mouse
{
private:
    string nama;
    string bentuk_mouse;
    string DPI;
    string warna;
    int jumlah_tombol;
    string tipe_koneksi;

public:
    Mouse() {}

    Mouse(string nama, string bentuk_mouse, string DPI, string warna, int jumlah_tombol, string tipe_koneksi)
    {
        this->nama = nama;
        this->bentuk_mouse = bentuk_mouse;
        this->DPI = DPI;
        this->warna = warna;
        this->jumlah_tombol = jumlah_tombol;
        this->tipe_koneksi = tipe_koneksi;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    void setBentukMouse(string bentuk_mouse)
    {
        this->bentuk_mouse = bentuk_mouse;
    }

    void setDPI(string DPI)
    {
        this->DPI = DPI;
    }

    void setWarna(string warna)
    {
        this->warna = warna;
    }

    void setJumlahTombol(int jumlah_tombol)
    {
        this->jumlah_tombol = jumlah_tombol;
    }

    void setTipeKoneksi(string tipe_koneksi)
    {
        this->tipe_koneksi = tipe_koneksi;
    }

    string getNama()
    {
        return nama;
    }

    string getBentukMouse()
    {
        return bentuk_mouse;
    }

    string getDPI()
    {
        return DPI;
    }

    string getWarna()
    {
        return warna;
    }

    int getJumlahTombol()
    {
        return jumlah_tombol;
    }

    string getTipeKoneksi()
    {
        return tipe_koneksi;
    }

    ~Mouse() {}
};